pkg/virt-config/config-map.go

```diff
@@ -41,21 +41,22 @@ import (
 )
 
 const (
-	configMapName             = "kubevirt-config"
-	FeatureGatesKey           = "feature-gates"
-	EmulatedMachinesKey       = "emulated-machines"
-	MachineTypeKey            = "machine-type"
-	useEmulationKey           = "debug.useEmulation"
-	ImagePullPolicyKey        = "dev.imagePullPolicy"
-	MigrationsConfigKey       = "migrations"
-	CpuModelKey               = "default-cpu-model"
-	CpuRequestKey             = "cpu-request"
-	MemoryOvercommitKey       = "memory-overcommit"
-	LessPVCSpaceTolerationKey = "pvc-tolerate-less-space-up-to-percent"
-	NodeSelectorsKey          = "node-selectors"
-	NetworkInterfaceKey       = "default-network-interface"
-	PermitSlirpInterface      = "permitSlirpInterface"
-	NodeDrainTaintDefaultKey  = "kubevirt.io/drain"
+	configMapName                     = "kubevirt-config"
+	FeatureGatesKey                   = "feature-gates"
+	EmulatedMachinesKey               = "emulated-machines"
+	MachineTypeKey                    = "machine-type"
+	useEmulationKey                   = "debug.useEmulation"
+	ImagePullPolicyKey                = "dev.imagePullPolicy"
+	MigrationsConfigKey               = "migrations"
+	CpuModelKey                       = "default-cpu-model"
+	CpuRequestKey                     = "cpu-request"
+	MemoryOvercommitKey               = "memory-overcommit"
+	LessPVCSpaceTolerationKey         = "pvc-tolerate-less-space-up-to-percent"
+	NodeSelectorsKey                  = "node-selectors"
+	NetworkInterfaceKey               = "default-network-interface"
+	PermitSlirpInterface              = "permitSlirpInterface"
+	PermitBridgeInterfaceOnPodNetwork = "PermitBridgeInterfaceOnPodNetwork"
```

Comment:
nit: to stay consistent with `permitSlirpInterface`, perhaps you should lower the case of the first letter - `permitBridgeInterfaceOnPodNetwork`.